client/analysisui/gridcheckbox.css

```diff
@@ -26,6 +26,14 @@
     margin-bottom: 0px;
 }
 
+.silky-option-checkbox .class[data-horizontal-align='center'] {
+    text-align: center;
+}
+
+.silky-option-checkbox .class[data-horizontal-align='right'] {
```

Comment:
wait, did you test this?

`.class` is almost definitely not what you want. that was just an example of where you would put a class.